packages/boxel/addon/components/boxel/input/selectable-token-amount/index.css

```diff
@@ -0,0 +1,64 @@
+.boxel-input-selectable-token-amount {
+  --boxel-input-selectable-token-amount-height: var(--boxel-sp-xl);
+  --boxel-input-selectable-token-amount-horizontal-padding: var(--boxel-sp-sm);
+  --boxel-input-selectable-token-amount-input-font-size: var(--boxel-font-size);
+
+  position: relative;
+  width: 100%;
+  font-family: var(--boxel-font-family);
+  font-size: var(--boxel-input-selectable-token-amount-input-font-size);
+}
+
+.boxel-input-selectable-token-amount .boxel-input-selectable-token-amount__input {
```

Comment:
ya… attaching an action to `onChange` that adjusts the input’s right padding seems like it would work. BUT I would like to defer this until it’s actually needed, I think for the initial use case it won’t be…???!?!?!